src/machine_learning.py

```diff
@@ -1,103 +0,0 @@
-# GeoFin : a python library for portfolio optimization and index replication

```

Comment:
it is not clear if the functionality of this file is moved to some other file or totally removed